pkg/apis/core/validation/validation.go

```diff
@@ -1999,10 +1998,8 @@ func ValidatePersistentVolumeUpdate(newPv, oldPv *core.PersistentVolume, opts Pe
 	}
 
 	// PersistentVolumeSource should be immutable after creation.
-	if !apiequality.Semantic.DeepEqual(newPv.Spec.PersistentVolumeSource, oldPv.Spec.PersistentVolumeSource) {
-		pvcSourceDiff := cmp.Diff(oldPv.Spec.PersistentVolumeSource, newPv.Spec.PersistentVolumeSource)
-		allErrs = append(allErrs, field.Forbidden(field.NewPath("spec", "persistentvolumesource"), fmt.Sprintf("spec.persistentvolumesource is immutable after creation\n%v", pvcSourceDiff)))
-	}
+	allErrs = append(allErrs, ValidateImmutableField(newPv.Spec.PersistentVolumeSource, oldPv.Spec.PersistentVolumeSource, field.NewPath("spec", "persistentvolumesource"))...)
```

Comment:
These errors seemed to have changed, is there something we can do to keep the errors the same as before?